Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/PersonDetailsViewModel.kt

```diff
@@ -70,7 +70,14 @@ class PersonDetailsViewModel @AssistedInject constructor(
     }.asLiveData2()
 
     private suspend fun createUiState(personCertificates: PersonCertificates, isLoading: Boolean): UiState {
-        val priorityCertificate = personCertificates.highestPriorityCertificate
+        val priorityCertificate = try {
+            personCertificates.highestPriorityCertificate
+        } catch (e: NoSuchElementException) {
+            Timber.e(e)
+            events.postValue(Back)
+            return UiState(name = "", emptyList())
+        }
+
         val certificateItems = mutableListOf<CertificateItem>().apply {
```

Comment:
Not sure about this. We allow PersonCertificates to be created with an empty list of certificates, which will always result in a crash as personIdentifier and highestPriorityCertificate are defined as not nullable. We could fix this underlying issue.

Reply:
- `PersonCertificates.personIdentifier` 
- `PersonCertificates.highestPriorityCertificate`

are now nullable